### PR TITLE
[1LP][RFR] Fix for host_drift_analysis

### PR DESCRIPTION
**Pull request by rlbabyuk**
Purpose or Intent
=================
{{pytest: -k test_host_drift_analysis -vvv --long-running --use-provider complete}}
Drift views is updated in https://github.com/ManageIQ/integration_tests/pull/6131

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 65a286eacecc8cd731b6ba4d4a741f69e81e59e1:

`cfme/tests/infrastructure/test_host_drift_analysis.py`:
- :bangbang: Line 43:27: **E231** *missing whitespace after ','*
- :bangbang: Line 58:101: **E501** *line too long (118 > 100 characters)*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit d5a24a87b14fad50e4f30b908695c9f6fd7a086f:

`cfme/infrastructure/host.py`:
- :bangbang: Line 285:26: **E251** *unexpected spaces around keyword / parameter equals*
- :bangbang: Line 285:28: **E251** *unexpected spaces around keyword / parameter equals*
- :bangbang: Line 285:40: **E251** *unexpected spaces around keyword / parameter equals*
- :bangbang: Line 285:42: **E251** *unexpected spaces around keyword / parameter equals*
- :bangbang: Line 285:59: **E251** *unexpected spaces around keyword / parameter equals*
- :bangbang: Line 285:61: **E251** *unexpected spaces around keyword / parameter equals*

`widgetastic_manageiq/__init__.py`:
- :heavy_exclamation_mark: Line 3763:31: **W292** *no newline at end of file*

Hmmm
:scream: You might like to check the lint of your code :point_up: 


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 1a392aee89cd8e6400ec23ea8ffa823e8928c670:

`widgetastic_manageiq/__init__.py`:
- :heavy_exclamation_mark: Line 3763:31: **W292** *no newline at end of file*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

File: cfme/tests/infrastructure/test_host_drift_analysis.py [hunk 1]
```diff
@@ -40,109 +37,83 @@ def pytest_generate_tests(metafunc):
...
+@pytest.fixture(scope='module')
+def set_host_credentials(provider, a_host, setup_provider_modscope):
+    host_list = provider.hosts
+    host_names = [host.name for host in host_list]
+    for host_name in host_names:
+        host_data = [host for host in host_list if host.name == host_name][0]
+        a_host.update_credentials_rest(credentials=host_data.credentials)
+
+    yield
+    a_host.update_credentials_rest(credentials=Host.Credential(principal="", secret=""))
```
Comment (quarckster) [inline cfme/tests/infrastructure/test_host_drift_analysis.py]: **QUESTION**
I still don't understand the point to not use https://github.com/ManageIQ/integration_tests/blob/master/cfme/utils/hosts.py#L21

**Pull request merged**